deployment/stacks/api.ts

```diff
@@ -202,6 +235,20 @@ export function createApiComponents(props: CreateApiComponentsProps) {
     ...commonProps,
   });
 
+  new LambdaDynamoEventSource(scope, "postKafkaData", {
+    entry: "services/app-api/handlers/kafka/post/postKafkaData.ts",
+    handler: "handler",
+    timeout: Duration.seconds(120),
+    memorySize: 2048,
+    retryAttempts: 2,
+    vpc,
+    vpcSubnets: { subnets: kafkaAuthorizedSubnets },
+    securityGroups: [kafkaSecurityGroup],
+    ...commonProps,
+    environment: { topicNamespace: "", ...commonProps.environment },
```

Comment:
@JonHolman @angelaco11 does this need namespace config per env? I am blanking on seeing where it is actually configured per branch